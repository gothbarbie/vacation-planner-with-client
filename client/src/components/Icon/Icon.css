.margin-right {
  margin-right: .5rem;
}

.margin-left {
  margin-left: .5rem;
}
