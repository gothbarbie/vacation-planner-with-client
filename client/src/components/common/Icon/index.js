// @flow

import React from 'react'
import classnames from 'classnames'

import './Icon.css'

type props = {
  name: string,
  isBrand?: boolean,
  size?: string,
}

export default ({ name, isBrand, size }: props) => (
  <i
    className={classnames(`icon fa-${name}`, {
      fab: isBrand,
      fas: !isBrand,
      'fa-xs': size === 'xs',
      'fa-sm': size === 'sm',
      'fa-lg': size === 'lg',
      'fa-2x': size === '2x',
      'fa-3x': size === '3x',
      'fa-5x': size === '5x',
      'fa-7x': size === '7x',
      'fa-10x': size === '10x',
    })}
  />
)
