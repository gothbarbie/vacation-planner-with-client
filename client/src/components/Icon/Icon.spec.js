import React from 'react'
import { shallow } from 'enzyme'

import Icon from './Icon'

describe('views/Schedule', () => {
  const icon = shallow(<Icon />)

  it('renders', () => {
    expect(icon).toMatchSnapshot()
  })
})
