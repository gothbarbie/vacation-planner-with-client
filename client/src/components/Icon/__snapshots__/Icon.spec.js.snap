// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views/Schedule renders 1`] = `
<i
  className="icon fa-undefined fas"
/>
`;
